spec/models/vm_migrate_workflow_spec.rb

```diff
@@ -58,6 +58,9 @@
       expect(request.userid).to eq(admin.userid)
       expect(request.requester_name).to eq(admin.name)
 
+      # workflow objects should be identical except the instance id
+      expect(request.workflow.inspect.split(' ')[1..-1]).to eq(workflow.inspect.split(' ')[1..-1])
```

Comment:
@bzwei I was thinking more like this: `expect(request.workflow).to be_a(described_class)`